src/helpers/utils.ts

```diff
@@ -152,21 +151,25 @@ export async function getVerifiedSubscriptions(batchSize = 1000) {
   return results;
 }
 
-export async function getAddressSubscriptions(address: string): Promise<TemplateId[]> {
-  const subscriptions = await db.queryAsync(
-    'SELECT email, subscriptions from subscribers WHERE address = ? AND verified > 0 LIMIT 1',
-    [address]
-  );
+export async function getSubscriber(address: string) {
+  const subscriber = (
+    await db.queryAsync(
+      'SELECT email, verified, subscriptions from subscribers WHERE address = ? ORDER BY created DESC, verified DESC LIMIT 1',
+      [address]
+    )
+  )[0];
 
-  if (!subscriptions[0]) {
+  if (!subscriber) {
     throw new Error('RECORD_NOT_FOUND');
```

Comment:
could we return a 200 response for this case? Because on FE we will see errors in the console, that is not right, because this is not an error, but a negative result.